spec/services/builders/external_pull_request_spec.rb

```diff
@@ -6,39 +6,59 @@
   context 'when there is already a pull request' do
     let!(:pull_request) { create(:external_pull_request) }
 
-    let(:pull_request_data) do
+    let(:pull_request_event_data) do
```

Comment:
Again, we could use the factory `gitub_api_client_pull_requests_events_payload` here. Maybe we could add to it a `pull_request` as a transient attribute?